openwisp_radius/admin.py

```diff
@@ -89,7 +89,7 @@ class RadiusCheckAdmin(MultitenantAdminMixin, TimeStampedEditableAdmin):
         'created',
         'modified',
     ]
-    autocomplete_fields = ['user']
+    autocomplete_fields = ('user', 'organization')
```

Comment:
Now that https://github.com/openwisp/openwisp-users/pull/339 is merged, we should be able to avoid this change, please try again and double check.